src/modules/fundingManager/bondingCurve/abstracts/RedeemingBondingCurveBase_v1.sol

```diff
@@ -270,6 +263,28 @@ abstract contract RedeemingBondingCurveBase_v1 is
             revert Module__BondingCurveBase__InsufficientOutputAmount();
         }
 
+        // ------------------------------------------------------------
+        // Interactions
+
+        // Burn issued token from user
+        _burn(_msgSender(), _depositAmount);
+
+        // Process the protocol fee. We can re-mint some of the burned tokens, since we aren't paying out
+        // the backing collateral
```

Comment:
```suggestion
        // Process the protocol fee on incoming issuance tokens. We can
        // re-mint since we aren't paying out the backing collateral.
```